ExaFunctions/CsvBulkImportFromBlobStorage.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Text;
+using Exasol.Utilities;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Exasol
+{
+    public static class CsvBulkImportFromBlobStorage
+    {
+        [FunctionName("CsvBulkImportFromBlobStorage")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                log.LogInformation("C# HTTP trigger function processed a request.");
```

Comment:
I have problems understanding this message. Can you please rephrase it?

Reply:
changed it to something way simpler